core/consensus/roundtimer_internal_test.go

```diff
@@ -113,10 +113,10 @@ func TestDoubleEagerLinearRoundTimer(t *testing.T) {
 }
 
 func TestGetTimerFunc(t *testing.T) {
```

Comment:
Do we test if timerFunc ever returns something that isn't `timerEagerDoubleLinear`?